### DISCUSSION TITLE
[select] 下拉选项展示问题

### DISCUSSION THREAD

**Discussion by lvfangren** [Tencent/tdesign-vue#815] [General]
### tdesign-vue 版本

0.40.2

### 重现链接

https://codesandbox.io/s/tdesign-vue-demo-forked-dl6vgi?file=/src/demo.vue

### 重现步骤

初始时options给空数组，设置onSearch动态获取（option文案超过select的宽度），此时option展示.....
<img width="345" alt="image" src="https://user-images.githubusercontent.com/25928863/164702404-7cd4cacf-96b6-4ffc-b7ba-58dfd8ed5f49.png">

初始时options给不为空数组，option文案超过select的宽度，此时option按最长文案宽度展示
<img width="497" alt="image" src="https://user-images.githubusercontent.com/25928863/164702480-20e4f905-fbad-4a5d-9682-8b8ddf95e5bf.png">

### 期望结果

超过select宽度下拉的option展示保持统一（显示省略或者全部展示）

### 实际结果

option超过select宽度时展示不统一

### 框架版本

Vue(2.6.11)

### 补充说明

我这边的场景是动态级联下拉，2号下拉的数据依赖1号下拉结果进行异步请求展示。

**Comment by uyarn** [comment]
你需要的应该可以通过加一行popupProps做到  设置一下popup的width 
`:popupProps="{ overlayStyle: { width: '300px' } }"`
refer to https://codesandbox.io/s/tdesign-vue-demo-forked-hzgneg?file=/src/demo.vue:177-234
<img width="1010" alt="image" src="https://user-images.githubusercontent.com/26377630/165668373-d35b342b-a734-4017-98b9-93a5e5f42db0.png">
